src/ontology/mondo-ingest.Makefile

```diff
@@ -605,8 +605,8 @@ tmp/%-synonyms-scope-type-xref.tsv: $(COMPONENTSDIR)/%.owl
 # todo: we may remove this output later output for analysis during development; at the end, remove it and its usages
 INPUT_FILES := $(wildcard tmp/synonym_sync_combined_cases_*.tsv)
 $(SYN_SYNC_DIR)/synonym_sync_combined_cases.robot.tsv: $(foreach n,$(ALL_COMPONENT_IDS), $(SYN_SYNC_DIR)/$(n)-synonyms.added.robot.tsv)
-	@head -n 2 $(firstword $(INPUT_FILES)) > $@
-	@for file in $(INPUT_FILES); do \
+	head -n 2 $(firstword $(INPUT_FILES)) > $@
```

Comment:
@matentzn I think you were going to run a `mondo-ingest` build. If you haven't yet, might be good to unsilence this just in case you get this issue:
- #691 

then we can see what's going on